robowflex_dart/CMakeLists.txt

```diff
@@ -11,13 +11,14 @@ set(CMAKE_MODULE_PATH
 include(CompileOptions)
 include(HelperFunctions)
 
-find_package(DART 6 REQUIRED
```

Comment:
Looking at this change, it should still be outside the conditional, but instead of REQUIRED it should be QUIET.

Reply:
Cool, moved this back and changed to QUIET.